docs/guides/lint.md

```diff
@@ -3,75 +3,235 @@ redirectFrom:
 :::
 
-Read about the [`--config` option](../commands/index.md) to use other file names or locations.
+:::success
+Omit this section if you don't have custom plugins.
+:::
 
-From a high-level, there are a few sub-sections.
+#### Examples
 
-```yaml
+```yaml single value
 lint:
   plugins:
-    # This section is where you can import local plugins.
-    # We don't support community plugins.
-    # You don't need to import our built-in plugins and rules.
-    # Omit this section if you don't have custom plugins.
     - './local-plugin.js'
+```
+
+```yaml multiple values
+lint:
+  plugins:
+    - ['./local-plugin.js', './another-local-plugin.js']
+```
+
+### Extends
 
+Use this section to choose the base configuration for further extension or adding your own plugins. You may override specific settings in the subsequent sections.
+
+* **type**: `array of strings`
+* **default**: `recommended`
+* **possible values**: `minimal`, `recommended`, `all`
+
+:::info
+Possible values specified above are related to built-in rules only. Custom plugins can contain additional configurations, so it can be as follows:
+
+```yaml
+lint:
+  extends:
+    - recommended
+    - my-custom-super-ruleset
+```
+
+Find more information in the [Configs in plugins](../resources/custom-rules/#configs-in-plugins) section.
+
+:::
+
+#### Examples
+
+```yaml single value
+lint:
   extends:
-    # This section is where you choose the base configurations.
-    # You may override specific settings in the subsequent sections.
-    - recommended # This is the default (and built in) configuration. If it is too strict, try `minimal`.
+    - minimal
+```
+
+```yaml multiple values
+lint:
+  extends:
+    - minimal
+    - recommended
+```
+
+### Resolve
+
+Use this section to specify external links in your definition that are not publicly accessible (except for Redocly API registry links).
+
+* **type**: `object`
+
+Redocly automatically resolves any API registry link or public URL. However, you may want to resolve links that are not API registry links or publicly accessible.
+
+Currently, OpenAPI CLI only supports `http` headers. Only one `http` header per URL is supported in the `resolve` section:
+
+```yaml
+lint:
+  resolve:
+    http:
+      headers:
+        - # header configuration
+```
+
+#### Header configuration
+
+| Property      | Description | Examples |
+| ------------- | ----------- | -------- |
+| `matches`     | Glob match against the URL. | `https://api.example.com/**` or `https://api.example.com/*.json` |
+| `name`        | HTTP header name.           | `Authorization` or `X-API-KEY` |
+| `value`       | The value of the header. Mutually exclusive with `envVariable`. | `123-abc` |
+| `envVariable` | The name of the environment variable that contains the value of the header. Mutually exclusive with `value`. | `SECRET_KEY` |
+
+:::success
+It is recommeded to use environment variables where possible.
+:::
+
+#### Example
+
+```yaml multiple values
```

Comment:
```suggestion
```yaml
```

Reply:
This custom text in code samples is visible only when you have two or more consecutive code blocks - that's when the portal displays them as tabs, with this text as the tab title. It won't be visible in this case, so we can remove it